src/plugins/Show.tsx

```diff
@@ -0,0 +1,24 @@
+import * as _ from "lodash";
+import * as React from "react";
+import {homeDirectory, pluralize, resolveDirectory, resolveFile, mapObject} from "../utils/Common";
+import {existsSync, statSync} from "fs";
+import {PluginManager} from "../PluginManager";
+import {Job} from "../shell/Job";
```

Comment:
Please remove unused imports; you can check if you're find using `npm run lint`.